setup.py

```diff
@@ -73,8 +73,7 @@ def build_extension_from_pyx(pyx_path, extra_sources_paths=None):
 ]
 cython_exts = cythonize(cython_modules, quiet=True)
 
-install_requires = ['menpo>=0.7.6,<0.8',
-                    'cyrasterize>=0.3,<0.4',
```

Comment:
@patricksnape with the new CPU rasterizer `GLRasterizer` is already optional (we do an import check) => relax `setup.py` dep.